packages/calling-component-bindings/review/stable/calling-component-bindings.api.md

```diff
@@ -233,6 +237,12 @@ export const useCallingSelector: <SelectorT extends (state: CallClientState, pro
 // @public
 export const useDeviceManager: () => StatefulDeviceManager | undefined;
 
+// @beta (undocumented)
```

Comment:
beta interface shouldn't show in stable api

Reply:
done